test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Evaluation/AutoComputePayloadMetadataInJsonIntegrationTests.cs

```diff
@@ -512,6 +516,21 @@ static AutoComputePayloadMetadataInJsonIntegrationTests()
             Model.AddElement(function4);
         }
 
+        [Fact]
+        public void AATest()
+        {
+            StringBuilder stringBuilder = new StringBuilder();
+            StringWriter stringWriter = new StringWriter(stringBuilder);
+
+            using (var writer = XmlWriter.Create(stringWriter))
+            {
+                IEnumerable<EdmError> errors;
+                CsdlWriter.TryWriteCsdl(Model, writer, CsdlTarget.OData, out errors);
+            }
+
+            string modelAsString = stringBuilder.ToString();
+        }

```

Comment:
Is this test intended to be part of this PR?

Reply:
No, That's for my helper. I will remove it.